src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy

```diff
@@ -114,19 +114,15 @@ class SwitchSyncSpec extends HealthCheckSpecification {
                 def swProps = switchHelper.getCachedSwProps(it.dpId)
                 def amountFlowRules = 2 //INGRESS_REVERSE, INGRESS_FORWARD
                 def amountS42Rules = swProps.server42FlowRtt ? 1 : 0
-                def amountMultiTableSharedRules = 0
-                if (swProps.multiTable) {
-                    amountS42Rules = swProps.server42FlowRtt ? amountS42Rules + 2 : amountS42Rules
-                    amountMultiTableSharedRules += 1
-                }
+                amountS42Rules = swProps.server42FlowRtt ? amountS42Rules + 2 : amountS42Rules
                 /**
                  * s42Rules
                  * multi/single table: SERVER_42_FLOW_RTT_INGRESS_REVERSE
                  * multiTable: SERVER_42_FLOW_RTT_INPUT, SERVER_42_FLOW_RTT_TURNING_COOKIE, SERVER42_QINQ_OUTER_VLAN
                  * some rule is in defaultCookies (SERVER_42_FLOW_RTT_OUTPUT_VLAN_COOKIE)
                  */
 
-                def amountRules = amountFlowRules + amountS42Rules + amountMultiTableSharedRules + it.defaultCookies.size()
+                def amountRules = amountFlowRules + amountS42Rules + 1 + it.defaultCookies.size()
```

Comment:
Could you please add the comment that +1 is for the amountMultiTableSharedRules to avoid wondering what this +1 means?

Reply:
Returned variable with self-explaining name instead.